.env-example

```diff
@@ -1,2 +1,4 @@
 SECRET_KEY="this_IS_secr3t"
 DEVENV="docker"
+MAPIT_URL="https://mapit.mysociety.org/"
+MAPIT_API_KEY=""
```

Comment:
I wonder what the best way to handles this is, for Docker users. Perhaps `README.md` should be updated, to include an instruction about creating a `.env` file with your MapIt API key in it, before running `docker-compose up`?